File: NATSInternal.Blazor/Helpers/AvatarHelper.cs
namespace NATSInternal.Blazor.Helpers;

public class AvatarHelper
{
    public string GetDefaultAvatar(string fullName)
    {
        return $"https://ui-avatars.com/api/?name={fullName.Replace(" ", "+")}" +
            "&background=random&size=256";
    }
}

File: NATSInternal.Frontend/NATSInternal.Svelte/src/helpers/avatarHelper.ts
const avatarHelper = {
  getDefaultAvatarUrlByFullName(fullName: string): string {
    const computedFullName = fullName.replaceAll(" ", "+");
    return `https://ui-avatars.com/api/?name=${computedFullName}&background=random`;
  }
};

export function useAvatarHelper() {
  return avatarHelper;
}